includes/Specials/SpecialRequestSSL.php

```diff
@@ -221,6 +230,10 @@ public function onSubmit( array $data ) {
 			$this->sendNotifications( $data['reason'], $this->getUser()->getName(), $requestID, $data['target'] );
 		}
 
+		$domainCheckJob = new DomainCheckJob( ['requestID' => $requestID] );
+		$jobQueueGroup = $this->jobQueueGroupFactory->makeJobQueueGroup();
```

Comment:
This can use JobQueueGroup since we aren't passing to a foreign wiki, JobQueueGroup service is the same as JobQueueGroupFactory->makeJobQueueGroup(), but like mentioned before, should use JobSpecification for service injection here also.

Reply:
Just realized you mean to construct the job via JobSpecification, sorry.